[terra-clinical-header]Accessibility Updates (#792)
cerner/terra-clinical: packages/terra-clinical-header/src/terra-dev-site/doc/clinical-header/AccessibilityGuide.2.doc.mdx

import { Badge } from 'terra-clinical-header/package.json?dev-site-package';

<Badge />

# Accessibility Guide for Terra Clinical Header

> Clinical Header allows users to organise web pages using the six levels of section heading. Where `level={1} (<h1>)` is the highest and `level={6} (<h6>)` is the lowest heading level.

Reviewer: ```suggestion
Clinical Header allows users to organize web pages using the six levels of section heading with `level={1} (<h1>)` being the highest and `level={6} (<h6>)` being the lowest heading level.
```
